core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java

```diff
@@ -56,7 +56,7 @@
 	public static final String CAMEL_SPRING_BOOT_STARTER = "camel-spring-boot-starter";
 	public static final String CAMEL_WILDFLY = "org.wildfly.camel";
 	
-	public static final String CAMEL_VERSION_LATEST_COMMUNITY = "2.23.1";
+	public static final String CAMEL_VERSION_LATEST_COMMUNITY = "2.24.1";
```

Comment:
this is a separated goal than the 7.3.1 upgrade, please do it with a separate task and PR

Reply:
reverted